ee/modules/380-metallb/docs/EXAMPLES.md

```diff
@@ -136,3 +137,26 @@ curl -s -o /dev/null -w "%{http_code}" 192.168.199.101:8080
 ```
 
 {% endraw %}
+
+## Example of binding a MetalLB IP address to an Ingress controller
```

Comment:
It is an example for BGP mode, [we are working](https://github.com/deckhouse/deckhouse/pull/7752) on a new inlet, L2LoadBalancer. Not sure if we need IngressNginxController example here.